<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FuzzWidget</class>
 <widget class="QWidget" name="FuzzWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="fuzzParametersLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="requestsToSendLabel">
       <property name="text">
        <string>Requests to Send</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSpinBox" name="requestsToSendSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="sendDeaySpinBox">
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="millisecondsLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>ms</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="sendDelayLabel">
       <property name="text">
        <string>SendDelay</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="replyTableView"/>
   </item>
   <item>
    <widget class="QPushButton" name="fuzzCancelButton">
     <property name="text">
      <string>Fuzz</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
